src/accounts/User.java

```diff
@@ -0,0 +1,96 @@
+package accounts;
+
+/**
+ * This is the User class that gets user information.
+ * @author aswecker
+ */
+public class User implements Person{
+
+    private final static int recordid = 0;
+    private String fname = "";
+    private String lname = "";
+    public static final int userid = 0;
```

Comment:
This will cause all accounts to have the ID 0, which will cause all food and mood to be associated with account 0 instead of the real account number. You can make this private or private final and set it in the constructor.